framework_tvb/tvb/core/services/burst_service.py

```diff
@@ -33,20 +33,21 @@
 from datetime import datetime
 
 from tvb.adapters.datatypes.db.mapped_value import DatatypeMeasureIndex
-from tvb.adapters.datatypes.db.time_series import TimeSeriesIndex
-from tvb.adapters.datatypes.h5.time_series_h5 import TimeSeriesH5
```

Comment:
The remaining import from adapters.datatypes should also be avoided from the service package

Reply:
I will use h5.index_for_h5_file(result_filename)() in that case.